WcaOnRails/spec/support/database_cleaner.rb

```diff
@@ -1,22 +1,18 @@
 RSpec.configure do |config|
   config.before(:suite) do
-    DatabaseCleaner.clean_with(:truncation)
+    DatabaseCleaner.clean_with(:truncation, except: %w(Countries Continents Events Rounds Formats teams))
   end
 
   config.before(:each) do
     DatabaseCleaner.strategy = :transaction
   end
 
   config.before(:each, js: true) do
-    DatabaseCleaner.strategy = :truncation
+    DatabaseCleaner.strategy = :truncation, {except: %w(Countries Continents Events Rounds Formats teams)}
```

Comment:
It worked on some tests, but any tests using js it failed. I only added it after the full test suite failed.